PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/actions/CreateBlobContainer.java

```diff
@@ -31,7 +32,7 @@ public void actionPerformed(NodeActionEvent e) {
     		connectionString = StorageClientSDKManager.getConnectionString(((StorageNode) parent).getStorageAccount());
     	}
         CreateBlobContainerForm form = new CreateBlobContainerForm(PluginUtil.getParentShell(), connectionString);
-
+        if(parent instanceof StorageNode) { form.setSubscription(new SubscriptionDetail(((StorageNode)parent).getSubscriptionId(), null, null, true)); }
```

Comment:
unneccessary to do null check. `obj instanceof SomeClass` is `false` if `obj` is `null`.